pkg/api/v1/types.go

```diff
@@ -883,11 +883,14 @@ type HTTPGetAction struct {
 	// Number must be in the range 1 to 65535.
 	// Name must be an IANA_SVC_NAME.
 	Port intstr.IntOrString `json:"port"`
-	// Host name to connect to, defaults to the pod IP.
+	// Host name to connect to, defaults to the pod IP. You probably want to set
+	// "Host" in httpHeaders instead.
 	Host string `json:"host,omitempty"`
 	// Scheme to use for connecting to the host.
 	// Defaults to HTTP.
 	Scheme URIScheme `json:"scheme,omitempty"`
+	// Custom HTTP headers to set in the request.
```

Comment:
Can you explain the semantic if I have multiple values for a key?  Do I get the same headerline more than once?

```
httpHeaders:
    X-Custom-Thing: [foo, bar]
```